CHANGELOG.md

```diff
@@ -1,8 +1,63 @@
+## v0.25.0 (2024-02-06)
+
+> **Note:**
+>  Waku Filter v2 now has three additional configuration options
+> `--filter-max-peers-to-serve=1000` drives how many peers can subscribe at once and
+> `--filter-max-criteria=1000` defines what is the maximum criterion stored per each peers
+>
+> This release introduces a major change in Filter v2 protocol subscription management.
+> From now each subscriber peer needs to refresh its living subscriptions by sending a SUBSCRIBER_PING message every 5 minutes by default, otherwise the peer's subscription will be removed.
+> `--filter-subscription-timeout=300` defines configurable timeout for the subscriptions (*in seconds*).
+
+
+### Features
+
+- running validators in /relay/v1/auto/messages/{topic} ([#2394](https://github.com/waku-org/nwaku/issues/2394)) ([e4e147bc](https://github.com/waku-org/nwaku/commit/e4e147bc))
+- **rln-relay-v2:** update C FFI api's and serde ([#2385](https://github.com/waku-org/nwaku/issues/2385)) ([b88facd0](https://github.com/waku-org/nwaku/commit/b88facd0))
+- running validators in /relay/v1/messages/{pubsubTopic} ([#2373](https://github.com/waku-org/nwaku/issues/2373)) ([59d8b620](https://github.com/waku-org/nwaku/commit/59d8b620))
+- shard aware relay peer management ([#2332](https://github.com/waku-org/nwaku/issues/2332)) ([edca1df1](https://github.com/waku-org/nwaku/commit/edca1df1))
+
+### Bug Fixes
+
+- bug fix in ci/Jenkinsfile.release: make -d:postgres part of NIMFLAGS ([#2395](https://github.com/waku-org/nwaku/issues/2395)) ([55d1b078](https://github.com/waku-org/nwaku/commit/55d1b078))
```

Comment:
I wouldn't include this point as this fixes something that I just broke the day before ;P
```suggestion
```